src/app/components/header/header.component.ts

```diff
@@ -1,10 +1,42 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrl: './header.component.scss'
 })
 export class HeaderComponent {
+  @Input() themeColor: 'black' | 'white' = 'white';
+  imagePaths: { black: { bx_menu_alt_right_image: string; carbon_favorite_image: string; shopping_card_image: string; user_image: string }; white: { bx_menu_alt_right_image: string; carbon_favorite_image: string; shopping_card_image: string; user_image: string } } = {
```

Comment:
create interface + move value to constants instead of storing it in component

Reply:
Fixed